apps/ucp-ui/src/Routes.test.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { render, screen } from "./shared/test/testUtils";
+import Routes from "./Routes";
+import { API_KEYS_CARD_TITLE_TEXT } from "./ApiKeys/constants";
+
+const mockLogout = jest.fn();
+
+jest.mock("@auth0/auth0-react", () => ({
+  useAuth0: () => ({
+    logout: mockLogout,
+  }),
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  withAuthenticationRequired: (component: any): any => component,
+}));
```

Comment:
Is this used indirectly in the test? Why do you need to mock it?

Reply:
Rendering the children requires these. I improved it to use an __mock__